keps/0002-kep-moveengine-hld.md

```diff
@@ -0,0 +1,152 @@
+---
+kep-number: 1
+title: MoveEngine - High Level Design
+authors:
+  - "@vitta"
+owners:
+  - Uma Mukkara
+  - Mayank
+editor: 
+creation-date: 2019-05-16
+last-updated: 2019-05-19
+status: provisional
+see-also:
+  - KEP-None
+replaces:
+  - KEP-None
+superseded-by:
+  - KEP-None
+---
+
+# KubeMove - High Level Design
+
+
+
+## Table of Contents
+
+* [Summary](#summary)
+* [Motivation](#motivation)
+    * [Goals](#goals)
+    * [Non-Goals](#non-goals)
+* [Proposal](#proposal)
+    * [Implementation Details/Notes/Constraints](#implementation-detailsnotesconstraints)
+    * [Risks and Mitigations](#risks-and-mitigations)
+* [Design acceptance criteria](#design-acceptance-criteria)
+* [Implementation History](#implementation-history)
+
+## Summary
+
+From the KubeMove HLD KEP, KubeMoveEngine CRD defines the placeholders for the workflow for a given application. The CR is created by the user by invoking the spec that has the following details
+   - Application to which this CR applies
+   - Target location including the namespace. User can specify multiple target locations, in which case multiple KMovePair CRs get created.
+   - The Dynamic Data Mobilizer or DDM details
+   - schedule time to transfer data at regular interval
+   - Telemetry management
+
+`MoveEngine` controller that is watching this CR will deploy the KubeMove framework(`MoveEngine`, `DataSync` controller) at remote cluster to
```

Comment:
By `deploy` do you mean, MVEC will install the kubemove operator on the remote cluster or it will just create CRs assuming the operator is pre-deployed?

Reply:
This KEP-2 is specific to MoveEngine CR, which is one of various CRs of KubeMove.
Summary content till the spec is copied from KEP-1. It needs correction. Will do the same and update this PR.